site/blog/2024-02-07-community-call/README.md

```diff
@@ -0,0 +1,42 @@
+
+Evgeny also brought up that on top of  the Specs themselves, there are many non-functional things that need to be updated too. For example, the Specs navigation needs to be improved, with a clear structure of the metadata properties. Since all this non-functional stuff is still part of the Specs, and should therefore follow a defined process, it was proposed that this part would be led by OKFN, and reviewed at the end by working group members.
+
+The last thing that we discussed during the calls were `patterns`. The purpose of the `patterns` has not been super well defined within the Frictionless project, and it is now unclear to some. The idea at the time they were introduced, was for them to be best practices, but not refined enough to be part of the Specs quite yet. It was also a way to share a certain approach. But there was no clearly defined mechanism to promote the `patterns` to be part of the Specs.
+
+Since the approach with this Specs update is to simplify as much as possible and cut out what is not needed, it was decided to include the `patterns` in the Specs documentation (as some kind of Cook Book) and not in the Specs themselves, and rename them as their current name suggests a certain formality.
+
+# Join us in February!
+Next community call is on February 22<sup>tnd</sup>, join us to hear all the exciting news about the [Frictionless specs](https://specs.frictionlessdata.io/) update! 
+
+Do you have something you would like to present to the community at one of the upcoming calls? Let us know via [this form](https://forms.gle/AWpbxyiGESNSUFK2A), or come and tell us on our [community chat on Slack](https://join.slack.com/t/frictionlessdata/shared_invite/zt-17kpbffnm-tRfDW_wJgOw8tJVLvZTrBg)(also accessible via a [Matrix bridge](https://matrix.to/#/#frictionlessdata:matrix.okfn.org) if you prefer to use an open protocol).
+
+You can sign up for the call already [here](https://docs.google.com/forms/d/e/1FAIpQLSeuNCopxXauMkrWvF6VHqOyHMcy54SfNDOseVXfWRQZWkvqjQ/viewform?usp=sf_link). Do you want to share something with the community? Let us know when you sign up. 
+
+# Call Recording
+Here is the recording of the full call:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/38RZZ15tIJw?si=kAocKNufgAlG64NE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+#Thank you
+On a final note, we would like to thank all community members that joined the call and that keep all these discussions alive, and particularly tho those who are putting a lot of thinking and hard work into the specs working group. Without you, all of this would not be possible.
```

Comment:
```suggestion
On a final note, we would like to thank all community members that joined the call and that keep all these discussions alive, and particularly to those who are putting a lot of thinking and hard work into the specs working group. Without you, all of this would not be possible.
```